arangod/Replication2/ReplicatedLog/Algorithms.h

```diff
@@ -92,18 +92,16 @@ struct ParticipantStateTuple : implement_compare<ParticipantStateTuple> {
   [[nodiscard]] auto isForced() const noexcept -> bool;
   [[nodiscard]] auto isFailed() const noexcept -> bool;
 
-  friend auto operator<=(ParticipantStateTuple, ParticipantStateTuple) noexcept -> bool;
+  friend auto operator<=(ParticipantStateTuple const&, ParticipantStateTuple const&) noexcept -> bool;
```

Comment:
For C++20 this should become a spaceship, if I am not mistaken.

Reply:
I'll gladly do that myself once the C++20 branch is merged.

Reply:
And even a defaulted one!